google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/FieldValue.java

```diff
@@ -50,6 +51,7 @@ public class FieldValue implements Serializable {
 
```

Comment:
It's a good idea to increment the serialVersionUID.

Reply:
Done.